Essentials/src/main/java/com/earth2me/essentials/EssentialsUpgrade.java

```diff
@@ -145,6 +146,60 @@ public static void uuidFileConvert(final IEssentials ess, final Boolean ignoreUF
         ess.getLogger().info("To rerun the conversion type /essentials uuidconvert");
     }
 
+    public void convertStupidNames() {
+        if (doneFile.getBoolean("updateUsersLegacyPathNames", false)) {
+            return;
+        }
+
+        LOGGER.info("Attempting to migrate legacy path names to Configurate");
```

Comment:
```suggestion
        LOGGER.info("Attempting to migrate legacy userdata keys to Configurate");
```
Maybe clearer? Not sure, but at first glance it looked like this had something to do with filesystem paths.